packages/react-components/react-toast/src/components/Toaster/Toaster.test.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { render } from '@testing-library/react';
+import { Toaster } from './Toaster';
+import { isConformant } from '../../testing/isConformant';
+import { ToasterProps } from './Toaster.types';
+
+describe('Toaster', () => {
+  const testid = 'test';
+  isConformant<ToasterProps>({
+    Component: Toaster,
+    displayName: 'Toaster',
+    requiredProps: { 'data-testid': testid } as ToasterProps,
+    getTargetElement: result => result.getByTestId(testid),
+    disabledTests: [
+      'component-has-root-ref',
+      'component-handles-ref',
+      'component-has-static-classnames-object',
+      'component-handles-classname',
```

Comment:
Imperative toast triggering means there's no easy way to render the slots that have the className - disabling for now but open to suggestions